api/clients/microsoft_client.py

```
@@ -76,300 +77,352 @@ async def authenticate(self) ->bool:
-            async with session.get(url, headers=headers, params=params
-                ) as response:
+            async with session.get(url, headers=headers, params=params) as response:
                 if response.status == HTTP_OK:
                     return await response.json()
                 else:
                     error_text = await response.text()
-                    logger.error('Graph API request failed: %s' % error_text)
-                    raise Exception(
-                        f'Graph API request failed: {response.status}')
+                    logger.error("Graph API request failed: %s" % error_text)
+                    raise Exception(f"Graph API request failed: {response.status}")
```

Pylint (3.3.8)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)